construct.yaml

```diff
@@ -53,7 +51,7 @@ specs:
   - opensearch-dsl
   # FIXME: We need to pin MySQL as 8.0.28 dropped support for TLS v1.0 and v1.1
   # In principle MySQL v5.7.10 supports TLSv1.2 but it wasn't enabled in LHCb at least
-  - mysql-client =8.0.27
+  - mysql-client
```

Comment:
As I'll probably forget by the time this is merged, this should be discussed at a BiLD (and the outdated comment should be removed).